config.py

```diff
@@ -1,5 +1,5 @@
 import os
-from dmutils.status import enabled_since, get_version_label
```

Comment:
I get the "no unused imports" thing, but in general we do need this import in our config to seamlessly add and use new feature flags without having to remember to also add an import.  I guess it shouldn't introduce too much  friction for a dev, but it does seem like this adds to overhead rather than simplifying things.